packages/discovery-provider/src/queries/get_tracks.py

```diff
@@ -202,6 +202,10 @@ def _get_tracks(session, args):
             base_query = base_query.join(TrackWithAggregates.aggregate_track).order_by(
                 sort_fn(AggregateTrack.save_count), TrackWithAggregates.track_id
             )
+        elif sort_method == SortMethod.comments:
```

Comment:
We want to be able to sort tracks by comment count?

Reply:
yeah probably not